sklearn/cluster/_kmeans.py

```diff
@@ -149,7 +149,7 @@ def _kmeans_plusplus(X, n_clusters, x_squared_norms,
 # K-means batch estimation by EM (expectation maximization)
 
 def _tolerance(X, tol):
-    """Return a tolerance which is independent of the dataset."""
```

Comment:
Please revert this change because it is unrelated with the the issue that you want to do here.